.github/workflows/dkp-cli.yml

```diff
@@ -0,0 +1,38 @@
+name: Update dkp-cli docs
+on:
+  workflow_dispatch:
+    inputs:
+      dkp-version:
+        description: DKP version to generate the docs for, e.g. v2.3.1
+        type: string
+        required: true
+        default: v2.3.1
+      target-version:
+        description: Target documentation version
+        type: choice
+        default: "2.3"
+        options:
+          - "2.3"
+          - "2.2"
+
+jobs:
+  dkp-cli-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Update dkp-cli docs
+        run: |
+          ci/update-dkp-cli-docs.sh
+        env:
+          DKP_CLI_VERSION: ${{ inputs.dkp-version }}
+          TARGET_VERSION: ""
```

Comment:
Is this intentional? If it's true, you can remove the env var

Reply:
No, this was not intentional, fixed.